snuba/web/rpc/v1/endpoint_trace_item_attribute_names.py

```diff
@@ -65,6 +78,21 @@ def visit_TraceItemFilterWrapper(
             self.keys.add(trace_item_filter.comparison_filter.key.name)
 
 
+def _backwards_compatible_mapping_expr() -> Expression:
+    map_elems = []
+    for key, val in ATTRIBUTE_MAPPINGS.items():
+        map_elems.append(key)
+        map_elems.append(val)
```

Comment:
Are these `map_elems` intended to be used anywhere?

Reply:
good catch